bin/improver-nbhood-iterate-with-mask

```diff
@@ -58,7 +61,9 @@ def main():
                     'the extra dimension from the cube. There is also '
                     'an option to re-mask the output cube, so that after '
                     'neighbourhood processing, non-zero values are only '
-                    'present for unmasked grid points. ')
+                    'present for unmasked grid points. '
+                    'There is an exta option of collapsing the dimension we '
```

Comment:
Since re-masking and collapsing are mutually exclusive, maybe "an alternative option"?

Reply:
Amended.